darwin/dataset/upload_manager.py

```diff
@@ -18,6 +16,7 @@
 
 import requests
 from darwin.datatypes import PathLike
+from darwin.exceptions import UploadRequestError, UploadStage
```

Comment:
`UploadStage` is not an exception, to me it does not belong there. If we really want to move it, we could go for our types file

Reply:
Reverted changes